.dockerignore

```diff
@@ -1,7 +1,7 @@
 Dockerfile
-packages/relay/node_modules/
-packages/relay/tests/
-packages/server/node_modules/
-packages/server/tests/
-node_modules/
-tools/
\ No newline at end of file
+**/packages/relay/node_modules/
```

Comment:
Which package dirs does this refer to?

Reply:
I read this to mean the `node_modules` under the relay.  My approach was to modify what was already there to use the proper syntax.  The `packages/relay/node_modules/` was already listed in the .dockerignore file.